code/modules/projectiles/projectile.dm

```diff
@@ -13,7 +13,6 @@
 	resistance_flags = LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
 	var/def_zone = ""	//Aiming at
 	var/mob/firer = null//Who shot it
-	var/obj/item/ammo_casing/ammo_casing = null

```

Comment:
no longer used. Also not very useful since a lot can happen to the casing between the moment the proj is fired and the moment it hits something.